vispy/testing/image_tester.py

```diff
@@ -49,9 +49,9 @@
 from subprocess import check_call, CalledProcessError
 import numpy as np
 
-from ..ext.six import string_types
-from ..ext.six.moves import http_client as httplib
-from ..ext.six.moves import urllib_parse as urllib
+import http.client as httplib
```

Comment:
From what I can tell, this module only uses HTTPConnection from http.client and urlencode from urllib.parse, mind just changing these to import those directly?

Reply:
No problem with that